packages/analytics/analytics-chart/src/components/chart-types/TimeSeriesChart.vue

```diff
@@ -197,23 +191,6 @@ const { options } = composables.useLinechartOptions({
   dimensionAxesTitle: toRef(props, 'dimensionAxesTitle'),
 })
 
-const mutableData = computed(() => {
-  return {
-    ...props.chartData,
-    datasets: props.chartData.datasets.map((e, i) => {
-      if (Array.isArray(props.datasetColors)) {
-        e.backgroundColor = props.datasetColors[i % props.datasetColors.length]
-        e.borderColor = darkenColor(e.backgroundColor, 50)
-      } else if (e.rawDimension in props.datasetColors) {
-        e.backgroundColor = props.datasetColors[e.rawDimension]
-        e.borderColor = darkenColor(e.backgroundColor, 50)
-      }
-      e.fill = props.fill
-      return e
-    }),
-  }
-})
-
```

Comment:
This was all redundant as well, props are all reactive by default, my mistake from the initial implementation.

Reply:
Verified in the sandbox, colors and fill still update accordingly when the prop is updated.